guidelines.md

```diff
@@ -146,3 +147,11 @@ As an example, consider minimum and maximum length of the reservation:
 
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
+
+## Taxations
+
+Each enterprise is located in a specific [Tax environment](operations/configuration.md#tax-environment) that offers a list of applicable [Taxations](operations/configuration.md#taxation). The numeric value of the taxations are represented by [Tax rate codes](operations/configuration.md#tax-rate) that are accepted within the tax environment. 
+
+Instead of numeric tax rates such as `0.1`, use [Tax rate codes](operations/configuration.md#tax-rate) such as `AT-S` in order to represent and calculate the correct taxation for each accounting item in Mews. To obtain the applicable codes, first download the enterprise information with [Get configuration](../operations/configuration.md#get-configuration) to identify the Tax Environment, then filter for the applicable [Tax rate codes](operations/configuration.md#tax-rate) from all tax environment information retrieved via [Get all tax environments](operations/configuration.md#get-all-tax-environments). 
```

Comment:
- use codes of [tax rate]s
- download -> retrieve
- the last sentence starting "then filter for the applicable ..." deosn't make sense to me. I'd go with something like "identify the [Tax Environment](...). You can find all information about the tax environments, taxations and tax rates   using [Get all tax environments](...)."